server/perkeepd/perkeepd.go

```diff
@@ -96,9 +96,9 @@ var (
 	flagHelp       = flag.Bool("help", false, "show usage")
 	flagLegal      = flag.Bool("legal", false, "show licenses")
 	flagConfigFile = flag.String("configfile", "",
-		"Config file to use, relative to the Perkeep configuration directory root. "+
-			"If blank, the default is used or auto-generated. "+
-			"If it starts with 'http:' or 'https:', it is fetched from the network.")
+		"Config file to use, relative to the Perkeep configuration directory root specified by this environnement variable CAMLI_CONFIG_DIR defled to $HOME/.config/perkeep. "+
+			"If the value of -configfile is blank, the default is used or auto-generated. "+
+			"If the value of -configfile is starts with 'http:' or 'https:', it is fetched from the network.")
```

Comment:
Why add words here? It was fine as-is.

Reply:
For the line 100-101.
I reread and there is apparent conflict of interpretation with the variable `CAMLI_CONFIG_DIR` and the configuration of `configfile`. I could remove it. You will see the difference.

Reply:
Provided that you use the wording that I have suggested, it will be fine.